app/scripts/modules/core/src/application/nav/verticalNav.less

```diff
@@ -92,3 +112,33 @@
     border-top: 1px solid var(--color-white);
   }
 }
+
+.collapsed {
```

Comment:
Careful, a top level selector named `collapsed` seems likely to collide with something else.

Reply:
I will rename with something more specific